.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: objective-c
+osx_image: xcode10.1
+
+before_install:
+  - gem install cocoapods -v '1.6.1'
+
+install: 
+  - pushd browser-sign-in
+  - pod install --repo-update
+  - popd
+  - pushd custom-sign-in
+  - pod install --repo-update
+  - popd
+
+script:
+  - set -o pipefail && xcodebuild -workspace browser-sign-in/OktaBrowserSignIn.xcworkspace -scheme OktaBrowserSignIn -destination platform\=iOS\ Simulator,OS\=12.1,name\=iPhone\ 5s build test | xcpretty
```

Comment:
Just a side note - we probably have to investigate how to "ask" Travis to build only certain projects. If I touched BrowserSignIn I don't want to build NativeSignIn project...

Reply:
Sorry, not understand. This repo contains two projects and we need to test both of them. Or?

Reply:
End-to-end tests are very time consuming. If I did some small change in projectN I don't want to trigger tests for project1...projectN-1 projects. It could be a cool enhancement for the CI builds but pretty sure can't be easily implemented with Travis :(

Reply:
Well, tests are not as time consuming as Travis's queue waiting, VM preparations, pods update, etc. So there wouldn't be any noticeable difference anyway. In such case it is better to run tests locally.

Reply:
Right, it depends on number of projects and tests. In our current state where we have small number of projects and tests it is totally fine to wait 10/15 minutes. In bigger projects end-to-end and UI tests can take hours